readme.md

```diff
@@ -240,6 +240,29 @@ ow(1, 'input', ow.number.validate(value => ({
 
 This can be useful for creating your own reusable validators which can be extracted to a separate npm package.
 
+#### catching(fn)
+
+Use a custom validation function that throws error. The function should throw an error if value is invalid or return nothing when value is valid. This allows to use specific validators for complex property or array types.
```

Comment:
Not clear whether it support any error or only an Ow ArgumentError. If both, is there any different behavior.

Reply:
I would say let's support only the ow one. This way we are not swallowing anyones errors.